jobQueue/jobWrapper.js

```diff
@@ -229,8 +229,8 @@ function createPromisifiedJob(jobName, data, req) {
  */
 function createJob(jobName, data, req) {
   const jobPriority = calculateJobPriority(jobName, data, req);
-  const job = jobQueue.create(jobName, data)
-  .ttl(TIME_TO_LIVE)
+  const job = jobQueue.create(jobName, data);
+  job.ttl(TIME_TO_LIVE)
```

Comment:
This was causing errors for me. I wasn't able to figure out why, but this fixed the problem.

I've been running it locally. Can you tell me how to do that?